src/main/java/seedu/duke/command/flightcommand/ListFlightCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.duke.command.flightcommand;
+
+import seedu.duke.command.Command;
+import seedu.duke.operationlist.OperationList;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+import java.util.logging.LogManager;
+import java.util.logging.Level;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+
+public class ListFlightCommand extends Command {
+
+    private static final Logger LOGGER = Logger.getLogger(ListFlightCommand.class.getName());
+
+    public static void setupLogger() {
+        LogManager.getLogManager().reset();
+        ListFlightCommand.LOGGER.setLevel(Level.ALL);
+        ConsoleHandler consoleHandler = new ConsoleHandler();
+        consoleHandler.setLevel(Level.SEVERE);
+        ListFlightCommand.LOGGER.addHandler(consoleHandler);
+        createFileHandler();
+        ListFlightCommand.LOGGER.log(Level.INFO, ui.getLoggerStartUpMessage());
+    }
```

Comment:
This method seems to be repeating for every class. Consider putting the logic up into a parent class that so that it is a common method shared by the child classes.